app/scripts/directives/logViewer.js

```diff
@@ -149,15 +160,15 @@ angular.module('openshiftConsole')
             };
 
             var fillHeight = function(animate) {
-              var content = $('.log-view-output');
+              var content = $("#" + $scope.logViewerID + ' .log-view-output');
               var contentTop = content.offset().top;
               if (contentTop < 0) {
                 // Content top is off the page already.
                 return;
               }
 
-              var fill = Math.floor($(window).height() - contentTop);
-              if (!$scope.chromeless) {
+              var fill = $scope.fixedHeight ? $scope.fixedHeight : Math.floor($(window).height() - contentTop);
```

Comment:
Would it easier to set the fixed height styles once in CSS and only call `fillHeight()` when not fixed?